### PR TITLE
Migrate Updates.js to styled components

### PR DESCRIPTION
**Pull request by SudharakaP**
Related to https://github.com/opencollective/opencollective/issues/3289

# Screenshots

<details>
<summary>Desktop</summary>

image
</details>

<details>
<summary>Mobile</summary>

Screen Shot 2020-11-15 at 23 21 34
</details>

### CODE REVIEW

File: components/Updates.js [hunk 1]
```diff
@@ -2,11 +2,32 @@ import React from 'react';
...
 import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
 
 import colors from '../lib/constants/colors';
 
+import Container from './Container';
 import StyledUpdate from './StyledUpdate';
 
+const LoadingContainer = styled.div`
+  color: ${colors.darkgray};
```
Comment (Betree) [inline components/Updates.js:13]: `lib/constants/colors.js` is deprecated, let's take this as an opportunity to remove it from this file.

```suggestion
  color: #797D7F;
```

Comment (SudharakaP) [reply components/Updates.js]: Done. 👍🏽

**Comment by Betree** [comment]
@SudharakaP prettier is complaining, could you check?

**Comment by SudharakaP** [comment]
> @SudharakaP prettier is complaining, could you check?

Ah, I've missed running prettier I think. Fixed it. 👍🏽

**Review by Betree — APPROVED** [review]
:+1:

**Pull request merged**